fairlearn/metrics/_generated_metrics.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import sklearn.metrics as skm
+
+from ._extra_metrics import (
+    true_positive_rate, true_negative_rate,
+    false_positive_rate, false_negative_rate,
+    selection_rate,
+)
+from . import make_derived_metric
+
+METRICS_SPEC = [
+    # base metrics from _extra_metrics
+    (true_positive_rate, ["difference", "ratio"]),
+    (true_negative_rate, ["difference", "ratio"]),
+    (false_positive_rate, ["difference", "ratio"]),
+    (false_negative_rate, ["difference", "ratio"]),
+    (selection_rate, ["difference", "ratio"]),
+
+    # base metrics from sklearn.metrics
+    (skm.accuracy_score, ["difference", "ratio", "group_min"]),
+    (skm.zero_one_loss, ["difference", "ratio", "group_max"]),
+    (skm.balanced_accuracy_score, ["group_min"]),
+    (skm.precision_score, ["group_min"]),
+    (skm.recall_score, ["group_min"]),
+    (skm.roc_auc_score, ["group_min"]),
+    (skm.mean_absolute_error, ["group_max"]),
+    (skm.mean_squared_error, ["group_max"]),
+    (skm.r2_score, ["group_min"]),
+    (skm.f1_score, ["group_max"]),
+    (skm.log_loss, ["group_min"]),
```

Comment:
the last two lines are buggy and don't make sense as fairness metrics! i know that it's not part of this PR, but can we fix?
```
    (skm.f1_score, ["group_min"]),
    (skm.log_loss, ["group_max"]),
```
The rationale being that the worst-case score is `group_min` and the worst-case loss is `group_max`.

Reply:
Well, I'd say getting those two the right way around is part of this :-)